docs/tr/docs/features.md

```diff
@@ -148,41 +147,41 @@ FastAPI'ın inanılmaz derecede kullanımı kolay, fakat inanılmaz derecede gü
-* 100% <abbr title="Python type annotations, with this your editor and external tools can give you better support">typeları belirtilmiş</abbr> codebase.
+* 100% <abbr title="Kodun ne kadarının test edildiği">testle kapsanmış</abbr>.
+* 100% <abbr title="Python tip belirteçleri bazı kod editörleri ve araçlarda daha iyi destek sunar">tipleri belirtilmiş</abbr> kod tabanı.
 * FastAPI ile yapılan bir çok proje insanlar tarafından kullanılıyor.
 
-## Starlette özellikleri
+## Starlette Özellikleri
 
 **FastAPI**, <a href="https://www.starlette.io/" class="external-link" target="_blank"><strong>Starlette</strong></a> ile tamamiyle uyumlu ve üzerine kurulu. Yani FastAPI üzerine ekleme yapacağınız herhangi bir Starlette kodu da çalışacaktır.
 
-`FastAPI` aslında `Starlette`'nin bir sub-class'ı. Eğer Starlette'nin nasıl kullanılacağını biliyor isen, çoğu işlevini aynı şekilde yapıyor.
+`FastAPI` aslında `Starlette`'nin alt katmanında kullanan bir üst framework. Eğer Starlette'nin nasıl kullanılacağını biliyor isen, çoğu işlevini aynı şekilde yapıyor.
 
 **FastAPI** ile beraber **Starlette**'nin bütün özelliklerine de sahip olacaksınız (FastAPI aslında Starlette'nin steroid basmış hali):
 
-* Gerçekten etkileyici bir performansa sahip.Python'un ise en hızlı frameworklerinden bir tanesi, <a href="https://github.com/encode/starlette#performance" class="external-link" target="_blank">**NodeJS** ve **Go** ile ise eşdeğer performansa sahip.</a>.
+* Gerçekten etkileyici bir performansa sahip. Python'un ise en hızlı frameworklerinden bir tanesi, <a href="https://github.com/encode/starlette#performance" class="external-link" target="_blank">**NodeJS** ve **Go** ile ise eşdeğer performansa sahip.</a>.
 * **WebSocket** desteği.
 * **GraphQL** desteği.
-* Kullanım halinde arka plan işlevleri.
-* Başlatma ve kapatma eventleri(startup and shutdown).
-* Test sunucusu HTTPX üzerine kurulu.
+* Çalışma anında arka planda iş  parçacıkları çalıştırabilme imkanı.
+* Başlatma ve kapatma anında kod çalıştırabilme(startup and shutdown).
+* HTTPX kütüphanesi üzerine kurulu test sistemi sunar.
 * **CORS**, GZip, Static dosyalar, Streaming responseları.
 * **Session and Cookie** desteği.
-* 100% test kapsayıcılığı.
-* 100% typeları belirtilmiş codebase.
+* Bütün kod tabanı 100% test ile kapsanmıştır.
+* Bütün kod tabanı 100% tip belirteçleriyle desteklenmiştir.
 
-## Pydantic özellikleri
+## Pydantic Özellikleri
 
-**FastAPI** ile <a href="https://pydantic-docs.helpmanual.io" class="external-link" target="_blank"><strong>Pydantic</strong></a> tamamiyle uyumlu ve üzerine kurulu. Yani FastAPI üzerine ekleme yapacağınız herhangi bir Pydantic kodu da çalışacaktır.
+**FastAPI**  <a href="https://pydantic-docs.helpmanual.io" class="external-link" target="_blank"><strong>Pydantic</strong></a> ile tamamen uyumlu ve üzerine kuruludur. Yani FastAPI üzerine ekleme yapacağınız herhangi bir Pydantic kodu da çalışacaktır.
```

Comment:
Yukarıda aynı yapı için `üzerine kurulu` burada ise `üzerine kuruludur` kullanılmış.

`Yani FastAPI üzerine ekleme yapacağınız herhangi bir Pydantic kodu da çalışacaktır.` => `Sonuç olarak sahip olduğunuz herhangi bir Pydantic kodu FastAPI ile birlikte çalışacaktır.`